vignettes/cookbook.Rmd

```diff
@@ -476,52 +476,74 @@ finalize_plot(
 
-* Similar to graphs above which needed special formatting on their axis labels, `scale_x_time()` is used to manage the range, breaks, and labels of the x-axis, specifically when that axis represents time. 
+- Similar to graphs above which needed special formatting on their axis labels, `ggplot2::scale_x_time()` is used to manage the range, breaks, and labels of the x-axis, specifically when that axis represents time. 
 
-* `lubridate::hms()` is used to convert a string into a time object. Lubridate is a very useful package if you need to manage dates and times. Learn more about it [here](https://r4ds.had.co.nz/dates-and-times.html).
+- `lubridate::hm()` converts strings into time period objects. Lubridate is a very useful package if you need to manage dates and times. Learn more about it [here](https://r4ds.had.co.nz/dates-and-times.html).
 
 
 #### Example code
 
 ```{r histogram, message=FALSE, warning=FALSE}
-# prepare an example dataset
-df <- read_csv("https://raw.githubusercontent.com/CMAP-REPOS/cmap_abm_report/master/data/csv37_tours_tod.csv")
-
-df <- df %>% 
-  filter(subgroup=="Model-Depart" | subgroup=="Model-Arrive") %>% 
-  filter(cat=="work" | cat=="school") %>% 
-  mutate(subgroup = factor(subgroup),
-         subgroup = fct_recode(subgroup, 
-                               "Departure" = "Model-Depart", 
-                               "Return" = "Model-Arrive"),
-         subgroup = fct_rev(subgroup),
-         cat = fct_rev(factor(cat)))
-
-# create the plot object
-p <- ggplot(df) +
-  geom_col(aes(x=hours, y=value, fill=cat)) +
-  facet_grid(cat~subgroup,
-             scale = "free_y") +
+# Get & format example dataset from ABM validation report GitHub repository
+df <- read_csv("https://raw.githubusercontent.com/CMAP-REPOS/cmap_abm_report/master/data/csv37_tours_tod.csv") %>%
+  filter(subgroup %in% c("Model-Depart", "Model-Arrive")) %>% 
+  filter(cat %in% c("work", "school", "discretionary")) %>% 
+  mutate(
+    purpose = fct_rev(factor(stringr::str_to_title(cat))),  # Title case
+    dep_arr = fct_rev(fct_recode(factor(subgroup), 
+                                 "Departure" = "Model-Depart", 
+                                 "Return" = "Model-Arrive"))
+  )
+
+# Create faceted histograms
+p <- ggplot(data = df, mapping = aes(x = hours, y = value)) +
+  
+  # Add histogram bars
+  geom_col(  # Or geom_histogram() if data is not already grouped
+    mapping = aes(fill = dep_arr)  # Color bars by tour purpose
+  ) +
+  
+  # Facet horizontally by tour purpose
+  facet_grid(rows = vars(purpose)) +
+  
+  # Apply CMAP theme and color palette
+  theme_cmap(
+    hline = 0,  # Add origin line at 0 for aesthetic reasons
+    gridlines = "hv",  # Show horizontal and vertical gridlines
+    strip.text = element_text(
+      hjust = 0.5  # Center facet labels at axis midpoint
```

Comment:
If there are lessons here such as the `hjust` change (or from @dlcomeaux's draft work with transit investments) about how to make `theme_cmap()` handle facets better than they do by default, I don't see an issue incorporating them into `theme_cmap_base()`. 

If we keep the adjustment here, I think it would be helpful to clearly comment that `strip.text` is a `...` arg that is passed on to `ggplot2::theme()`

Reply:
My only concern with shifting `hjust` to 0.5 in this case would be that, for labels on top of the graph (say it's a differently oriented facet), a left-aligned label is more consistent with the title and caption's labeling style.

Reply:
Yeah, I think this might be good to incorporate into `theme_cmap()`, but I would want a recommendation from Comms about preferred label placement first.

Reply:
> 
> 
> My only concern with shifting `hjust` to 0.5 in this case would be that, for labels on top of the graph (say it's a differently oriented facet), a left-aligned label is more consistent with the title and caption's labeling style.

`strip.text` is one of those `theme()` args that has many varieties: `strip.text.x|y` and maybe even `strip.text.x|y.left|right|top|bottom` (one of these is in `theme_cmap_base()`, which means I probably borrowed it from somewhere, but strangely it is not [listed as an arg](https://ggplot2.tidyverse.org/reference/theme.html).

But at least it is possible to have a different default for x and y strips.